ftplugin/julia/slime.vim

```diff
@@ -0,0 +1,8 @@
+function! _EscapeText_julia(text)
+	if len(split(a:text,"\n")) > 1
+		return ["begin\n", a:text, "end\n"]
+	else
+		return a:text
+	endif
+endfunction
```

Comment:
```suggestion
function! _EscapeText_julia(text)
  if match(a:text, "\n.") > -1
    return ["begin\n", a:text, "end\n"]
  end
  return a:text
endfunction
```

- changed if to only look for newline
- changed indent (2 spaces)
- no need for `else` on `if-return`